src/target/riscv/riscv.h

```diff
@@ -65,6 +65,13 @@ enum riscv_isrmasking_mode {
 	RISCV_ISRMASK_STEPONLY,
 };
 
+enum riscv_state {
```

Comment:
Cosmetic thing: This could be called `enum riscv_hart_state`.

```suggestion
enum riscv_hart_state {
```

Reply:
Will do.